modules/lazy_resampling_compose.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>A typo here.</p><p>"because the&nbsp;<code>RandCropByLabelClassesD</code>&nbsp;relies on the&nbsp;<code>label</code>&nbsp;inpute" should be:</p><p>"because the&nbsp;<code>RandCropByLabelClassesD</code>&nbsp;relies on the&nbsp;<code>label</code>&nbsp;input"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>